vector/src/main/java/im/vector/fragments/VectorRoomDetailsMembersFragment.java

```diff
@@ -543,6 +543,25 @@ private boolean isUserAdmin() {
         return isAdmin;
     }
 
+    /**
+     * Check whether the current user is allowed to invite other Tchap users.
+     *
+     * @return true if user is allowed to invite, false otherwise
+     */
+    private boolean isUserAllowedToInvite() {
+        boolean isAllowed = false;
+
+        if ((null != mRoom) && (null != mSession)) {
+            PowerLevels powerLevels;
+
+            if (null != (powerLevels = mRoom.getState().getPowerLevels())) {
```

Comment:
please avoid assignment in if condition. Better practice is to combine declaration and assignment.